{"uuid":"0978589c-7f28-4c57-b31f-f0727ea350bf","historyId":"1f56c0b9c7921cc7b19f0a2d08f27252","fullName":"APITest.loginWithRegisteredUser","labels":[{"name":"package","value":"APITest"},{"name":"testClass","value":"APITest"},{"name":"testMethod","value":"loginWithRegisteredUser"},{"name":"parentSuite","value":"Default Suite"},{"name":"suite","value":"OpenfitAPIs"},{"name":"subSuite","value":"APITest"},{"name":"host","value":"MC-0126"},{"name":"thread","value":"17550@MC-0126.main(1)"}],"links":[],"name":"Login with registered user.","status":"failed","statusDetails":{"known":false,"muted":false,"flaky":false,"message":"The following asserts failed:\n\texpected [testerlvv8d@yopmail.com] but found [email],\n\texpected [null] but found [given_name, family_name, sub, id_token, access_token, refresh_token]","trace":"java.lang.AssertionError: The following asserts failed:\n\texpected [testerlvv8d@yopmail.com] but found [email],\n\texpected [null] but found [given_name, family_name, sub, id_token, access_token, refresh_token]\n\tat org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:43)\n\tat Methods.getLoginResponseContent(Methods.java:147)\n\tat APITest.loginWithRegisteredUser_aroundBody8(APITest.java:91)\n\tat APITest$AjcClosure9.run(APITest.java:1)\n\tat org.aspectj.runtime.reflect.JoinPointImpl.proceed(JoinPointImpl.java:149)\n\tat io.qameta.allure.aspects.StepsAspects.step(StepsAspects.java:46)\n\tat APITest.loginWithRegisteredUser(APITest.java:86)\n\tat sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)\n\tat sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)\n\tat sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)\n\tat java.lang.reflect.Method.invoke(Method.java:498)\n\tat org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)\n\tat org.testng.internal.Invoker.invokeMethod(Invoker.java:583)\n\tat org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)\n\tat org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)\n\tat org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)\n\tat org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)\n\tat org.testng.TestRunner.privateRun(TestRunner.java:648)\n\tat org.testng.TestRunner.run(TestRunner.java:505)\n\tat org.testng.SuiteRunner.runTest(SuiteRunner.java:455)\n\tat org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)\n\tat org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)\n\tat org.testng.SuiteRunner.run(SuiteRunner.java:364)\n\tat org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)\n\tat org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)\n\tat org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)\n\tat org.testng.TestNG.runSuitesLocally(TestNG.java:1137)\n\tat org.testng.TestNG.runSuites(TestNG.java:1049)\n\tat org.testng.TestNG.run(TestNG.java:1017)\n\tat org.testng.IDEARemoteTestNG.run(IDEARemoteTestNG.java:73)\n\tat org.testng.RemoteTestNGStarter.main(RemoteTestNGStarter.java:123)\n"},"stage":"finished","start":1558359628067,"stop":1558359632621,"steps":[{"name":"Subscribe with valid product ID: {0}, Login with valid credentials: {1}","status":"failed","statusDetails":{"known":false,"muted":false,"flaky":false,"message":"The following asserts failed:\n\texpected [testerlvv8d@yopmail.com] but found [email],\n\texpected [null] but found [given_name, family_name, sub, id_token, access_token, refresh_token]","trace":"java.lang.AssertionError: The following asserts failed:\n\texpected [testerlvv8d@yopmail.com] but found [email],\n\texpected [null] but found [given_name, family_name, sub, id_token, access_token, refresh_token]\n\tat org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:43)\n\tat Methods.getLoginResponseContent(Methods.java:147)\n\tat APITest.loginWithRegisteredUser_aroundBody8(APITest.java:91)\n\tat APITest$AjcClosure9.run(APITest.java:1)\n\tat org.aspectj.runtime.reflect.JoinPointImpl.proceed(JoinPointImpl.java:149)\n\tat io.qameta.allure.aspects.StepsAspects.step(StepsAspects.java:46)\n\tat APITest.loginWithRegisteredUser(APITest.java:86)\n\tat sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)\n\tat sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)\n\tat sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)\n\tat java.lang.reflect.Method.invoke(Method.java:498)\n\tat org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)\n\tat org.testng.internal.Invoker.invokeMethod(Invoker.java:583)\n\tat org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)\n\tat org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)\n\tat org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)\n\tat org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)\n\tat org.testng.TestRunner.privateRun(TestRunner.java:648)\n\tat org.testng.TestRunner.run(TestRunner.java:505)\n\tat org.testng.SuiteRunner.runTest(SuiteRunner.java:455)\n\tat org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)\n\tat org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)\n\tat org.testng.SuiteRunner.run(SuiteRunner.java:364)\n\tat org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)\n\tat org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)\n\tat org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)\n\tat org.testng.TestNG.runSuitesLocally(TestNG.java:1137)\n\tat org.testng.TestNG.runSuites(TestNG.java:1049)\n\tat org.testng.TestNG.run(TestNG.java:1017)\n\tat org.testng.IDEARemoteTestNG.run(IDEARemoteTestNG.java:73)\n\tat org.testng.RemoteTestNGStarter.main(RemoteTestNGStarter.java:123)\n"},"stage":"finished","start":1558359628114,"stop":1558359632610,"steps":[{"name":"Make request to create free user: {0}","status":"passed","stage":"finished","start":1558359628115,"stop":1558359631501}]}],"parameters":[]}